src/CachePlugin.php

```diff
@@ -379,7 +379,12 @@ private function createResponseFromCacheItem(CacheItemInterface $cacheItem)
 
         /** @var ResponseInterface $response */
         $response = $data['response'];
-        $response = $response->withBody($this->streamFactory->createStream($data['body']));
+        $stream = $this->streamFactory->createStream($data['body']);
+        if ($stream->isSeekable()) {
```

Comment:
if the stream is not seekable then it's not usable, what do you think about throwing an exception if the stream is not rewindable ?